docs/send-data/hosted-collectors/microsoft-source/ms-azure-event-hubs-source.md

```diff
@@ -0,0 +1,81 @@
+---
+id: ms-azure-event-hubs-source
+title: Azure Event Hubs for Logs - Beta
+description: The Azure Event Hubs Source provides a secure endpoint to receive data from Azure Event Hubs.
+---
+
```

Comment:
Can we add the Beta tag, @kimsauce 

<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>

Reply:
@ajaiswals thank you so much - great catch!